runatlantis.io/blog.md

```diff
@@ -9,8 +9,9 @@ We are thrilled to have you here! Our blog is a collection of insightful article
 
 ### Explore Our Popular Posts
 
-We have a rich history of blog posts dating back to 2017-2019. Here are some of our popular posts:
```

Comment:
this is fine with 2019

once we have more blog posts, I am thinking to remove this section